android-components/components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksInterceptor.kt

```diff
@@ -190,9 +197,8 @@ class AppLinksInterceptor(
         @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
         internal var userDoNotInterceptCache: MutableMap<Int, Long> = mutableMapOf()
 
-        // This should be improved.  See https://bugzilla.mozilla.org/show_bug.cgi?id=1877323
         @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-        internal var lastHasExternalAppTimestamp: Long = 0L
+        internal var lastApplinksTimestamp: Pair<String?, Long> = Pair(null, 0L)
```

Comment:
I wonder if we shouldn't have a specific type to make it clearer? 

Or at least have a name reflect better its content, like `lastApplinksPackageWithTimestamp`?

Reply:
Sounds good to me!